### PR TITLE
Uranium added and Power Generator requires Uranium

### PR DESCRIPTION
**Pull request by wilcomega**
Same as the coal one except for it now uses uranium. Sprite made by me. #466

### CODE REVIEW

File: Assets/StreamingAssets/Data/Furniture.xml [hunk 1]
```diff
@@ -171,8 +171,14 @@
 
       <BuildingJob jobTime="0">
       </BuildingJob>
-
-	
+      
+      <Params>
+        <Param name="burnTime" value="0" />
```
Comment (zwrawr) [inline Assets/StreamingAssets/Data/Furniture.xml]: Should be 4 spaces not 2 in this indent, also looks like there's some tab's in this file

File: Assets/StreamingAssets/Images/Inventory/Uranium.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Sprites>
+	<Sprite name="Uranium"     x="0" y="0" w="16" h="16" pixelPerUnit="16" />
```
Comment (zwrawr) [inline Assets/StreamingAssets/Images/Inventory/Uranium.xml]: tab here

**Comment by wilcomega** [comment]
I fixed it, is there a difference between a tab and four spaces? because i cant see any aside from the binairy difference

**Comment by vogonistic** [comment]
Merged!

**Comment by ComputerOverlord** [comment]
So... It's a fission station

**Pull request merged**